### DISCUSSION TITLE
How to adjust OSD down timeout

### DISCUSSION THREAD

**Discussion by KKonak** [rook/rook#13975] [Q&A] [answered]
I want adjust the time it takes for a Ceph OSD to be marked down. During failover the OSD pod will go into pending state waiting for the server to come back online but when checking "ceph status" all OSDs are UP & IN. During this time "ceph osd status" cannot load and new PVCs cannot be created until ceph marks it as down. After about 30+ minutes it will finally mark down. The cluster is able to recover when the node rejoins, but having a 30 minute down period is not ideal.

What is the ceph config to adjust this timeout.

**Comment by travisn** [comment, marked as answer]
See the [OSD config topic](https://docs.ceph.com/en/latest/rados/configuration/osd-config-ref/). Perhaps you need to modify `osd_fast_fail_on_connection_refused`.

**Reply by KKonak** [reply]
We found `mon_osd_report_timeout` to work for us. It now fails over immediately, thanks for the reply! [Monitor/OSD interaction](https://docs.ceph.com/en/latest/rados/configuration/mon-osd-interaction/#configuration-settings)
